subprojects/docs/src/docs/userguide/img/java-test-configurations.dot

```diff
@@ -0,0 +1,37 @@
+digraph javaPluginConfigurations {
+
+    rankdir = BT;
+    node [fontname="Sans", shape=box, style=filled, fillcolor=chartreuse];
+    edge [fontname="Sans"];
+
+    /* tasks */
+    test [fillcolor=darkslategray1, fontname="Courier New"]
+    compileTestJava [fillcolor=darkslategray1, fontname="Courier New"]
+
+    /* deprecated confs */
+    compile [fontcolor=gray55]
+    runtime [fontcolor=gray55]
+    testCompile [fontcolor=gray55]
+    testRuntime [fontcolor=gray55]
+
+    testCompileClasspath [fillcolor=lightblue]
+    testRuntimeClasspath [fillcolor=lightblue]
+
+    /* configurations */
+    implementation -> compile;
+    runtime -> compile
+
+    testCompile -> compile
+    testImplementation -> {testCompile implementation}
+    testCompileClasspath -> {testCompile testCompileOnly testImplementation}
+    testRuntime -> {runtime testCompile}
+    testRuntimeOnly -> runtimeOnly
+    testRuntimeClasspath -> {testRuntimeOnly testRuntime testImplementation}
+
+    /* tasks */
+    test -> testRuntime [style=dashed label="uses"]
```

Comment:
I am not sure about this, but I think the test task uses the `testRuntimeClasspath`

Reply:
Good catch. Copy-paste error.